src/index.html

```diff
@@ -12,7 +15,39 @@
       href="styles/index.scss"
     />
```

Comment:
The href attribute should point to a .css file, not a .scss file. The browser does not understand SCSS syntax. Your SCSS should be compiled into CSS, and the CSS file should be linked here.